### ISSUE TITLE
HedgeModManager Crash When Loading Non-Existent XAML File

### ISSUE THREAD

**Issue by L0veL0st** [thesupersonic16/HedgeModManager#347]
HMM Info:
```
    Version: 7.6-1
    Args: 
    StartDir: C:\Users\rafio\Downloads\
    Process Level: User
    Game: Unnamed Game
    Update Status: Up To Date
```

Exception:
```
    Type: FileNotFoundException
    Message: Could not find file 'C:\Users\rafio\Downloads\Languages\Teamod_LNG_Dutch.xaml'.
    Source: mscorlib
    Function: Void WinIOError(Int32, System.String)
    StackTrace: 
       at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
       at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
       at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
       at HedgeModManager.HedgeApp.GetLanguageResource(String culture) in C:\Users\SuperSonic16\source\repos\HedgeModManager\HedgeModManager\HedgeApp.xaml.cs:line 415
       at HedgeModManager.HedgeApp.CountLanguages() in C:\Users\SuperSonic16\source\repos\HedgeModManager\HedgeModManager\HedgeApp.xaml.cs:line 426
       at HedgeModManager.HedgeApp.Main(String[] args) in C:\Users\SuperSonic16\source\repos\HedgeModManager\HedgeModManager\HedgeApp.xaml.cs:line 171
    InnerException: 
```

**Comment by RagdollClash** [comment]
Try moving HedgeModManager to another location, it seems like it's trying to load a language file for a different program.

**Comment by thesupersonic16** [comment]
I will look into this issue. Were you intending on translating HedgeModManager?

**Comment by thesupersonic16** [comment]
Issue should be fixed in 18ca58d85174caa2829eff6acd00481583eb7d9b. At the moment there are two solutions until we make the next release:

- Run the latest dev build of HedgeModManager which can be downloaded [here](https://nightly.link/thesupersonic16/HedgeModManager/workflows/build/rewrite/HedgeModManager-Release.zip). Please let me know if it continues to break with this.

OR
- Move HedgeModManager.exe into another location where there isn't a conflicting "Languages" folder

I will close this issue once we make the next release.
